config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  root 'pages#home'
```

Comment:
Just moving the `root` to the top. :)

And adding a route to the new episode page.